### PR TITLE
Update styleguide

### PR DESCRIPTION
**Pull request by aduggin**

### CODE REVIEW

File: app/assets/javascripts/consent.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+(function ($) {
+  'use strict';
+
+  window.ROG = window.ROG || {};
```
Comment (andrewgarner) [inline app/assets/javascripts/consent.js]: Should this be `window.output` now given the re-naming of the application?

File: app/views/styleguide/pages/input.html.erb [hunk 2]
```diff
@@ -1,10 +1,410 @@
...
+  <div class="form-group">
+    <fieldset>
+      <legend>
+        <span>Organisation:</span>
+      </legend>
+
+      <div class="radio">
+        <label for="appointment_summary_guider_organisation_cab">
+          <input class="t-guider-organisation-cab" type="radio" value="cab" name="appointment_summary[guider_organisation]" id="appointment_summary_guider_organisation_cab" />
+          Citizens Advice
```
Comment (andrewgarner) [inline app/views/styleguide/pages/input.html.erb]: Citizens Advice have been removed from the application.

File: app/views/styleguide/pages/input.html.erb [hunk 3]
```diff
@@ -1,10 +1,410 @@
...
-    <input id="name" type="text"/>
+<p class="lead">Use this form to enter the visitor or caller's personal details provided during
+  the guidance appointment and to issue a personalised record of guidance.</p>
+
+<div class="consent">
+  <p class="highlight highlight-orange">Has visitor consented to sharing data for purpose of creating and sending record of guidance?</p>
+</div>
+
+<div class="display_if_consent">
+  <h2>Visitor or caller details</h2>
```
Comment (andrewgarner) [inline app/views/styleguide/pages/input.html.erb]: They're always a caller.

File: app/views/styleguide/pages/input.html.erb [hunk 4]
```diff
@@ -1,10 +1,410 @@
...
-<h1><%= title 'Output builder' %></h1>
+<div class="page-header">
+  <h1><%= title 'New appointment summary' %></h1>
+</div>
 
-<h2>Personal Details</h2>
-  <div class="l-form-row">
-    <label for="name">Client's name:</label>
-    <input id="name" type="text"/>
+<p class="lead">Use this form to enter the visitor or caller's personal details provided during
```
Comment (andrewgarner) [inline app/views/styleguide/pages/input.html.erb]: They're always a caller.

File: app/views/styleguide/pages/input.html.erb [hunk 5]
```diff
@@ -1,10 +1,410 @@
...
+  </div>
+
+  <hr>
+
+  <h2>Pension pot details</h2>
+
+  <div class="form-group">
+    <fieldset>
+      <legend>
+        <span>Does visitor/caller have a Defined Contribution pension:</span>
```
Comment (andrewgarner) [inline app/views/styleguide/pages/input.html.erb]: They're always a caller.

File: app/views/styleguide/pages/input.html.erb [hunk 6]
```diff
@@ -1,10 +1,410 @@
...
+    <input type="submit" name="commit" value="Submit" class="btn-success t-submit btn-lg" />
+  </p>
+
+</div>
+
+<script type="text/html" id="consent">
+  <div class="checkbox">
+    <label for="appointment_summary_consent">
+      <input class="t-continue-working" type="checkbox" value="1" name="appointment_summary[consent]" id="appointment_summary_consent" />
+      Visitor/caller has consented to sharing data for purpose of creating and sending record of guidance
```
Comment (andrewgarner) [inline app/views/styleguide/pages/input.html.erb]: They're always a caller.

File: app/views/styleguide/pages/input.html.erb [hunk 7]
```diff
@@ -1,10 +1,410 @@
...
+    </label>
+  </div>
+</script>
+
+<script type="text/html" id="pension_pot_accuracy">
+  <div class="col-md-3 col-accuracy">
+    <div class="form-group">
+      <fieldset>
+        <legend>
+          <span>How accurate is the value given by the vistor/calller?</span>
```
Comment (andrewgarner) [inline app/views/styleguide/pages/input.html.erb]: They're always a caller.

Comment (andrewgarner) [inline app/views/styleguide/pages/input.html.erb]: And never a "calller"!

File: app/assets/javascripts/consent.js [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+  var consent = {
+    init: function () {
+      this.insertHTML();
+      this.cache();
+      this.bindEvents();
+    },
+    insertHTML: function () {
+      var html = document.getElementById('consent').innerHTML;
```
Comment (gjvis) [inline app/assets/javascripts/consent.js]: why the raw Javascript if we've already JQuery available?

File: vendor/assets/javascripts/tiny-pubsub.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
+/*
+ * jQuery Tiny Pub/Sub
+ * https://github.com/cowboy/jquery-tiny-pubsub
```
Comment (andrewgarner) [inline vendor/assets/javascripts/tiny-pubsub.js]: Can we pull this in with Bower?

File: app/assets/javascripts/pension-pot-accuracy.js [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+  window.ROG = window.ROG || {};
+
+  var pensionPotAccuracy = {
+    init: function () {
+      this.insertHTML();
+      this.cache();
+      this.bindEvents();
+    },
+    insertHTML: function () {
+      var options = document.getElementById('pension_pot_accuracy').innerHTML;
```
Comment (gjvis) [inline app/assets/javascripts/pension-pot-accuracy.js]: same question here - I think it'd be more consistent to use JQuery

File: app/assets/javascripts/pension-pot-input.js [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+  var pensionPotInput = {
+    init: function () {
+      this.insertHTML();
+      this.cache();
+      this.subscribe();
+      this.bindEvents();
+    },
+    insertHTML: function () {
+      var html = document.getElementById('pension_pot_input_error').innerHTML;
```
Comment (gjvis) [inline app/assets/javascripts/pension-pot-input.js]: Why the vanilla javascript?

File: app/assets/javascripts/pension-pot-preview.js [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+    accuracy: 'exact',
+    value1: '',
+    value2: '',
+    init: function () {
+      this.insertHTML();
+      this.cache();
+      this.subscribe();
+    },
+    insertHTML: function () {
+      var preview = document.getElementById('pension_pot_preview').innerHTML;
```
Comment (gjvis) [inline app/assets/javascripts/pension-pot-preview.js]: Why the vanilla javascript?

File: app/assets/javascripts/consent.js (hunk 1 above)
Comment (aduggin) [inline app/assets/javascripts/consent.js]: As discussed I'll make it PWO (pension wise output)

**Pull request merged**